lib/wanda_web/schemas/v3/catalog/check.ex

```diff
@@ -102,10 +106,32 @@ defmodule WandaWeb.Schemas.V3.Catalog.Check do
                 "Message returned when the check return value is warning. Only available for `expect_enum` expectations"
             }
           },
-          required: [:name, :type, :expression]
+          required: [:name, :type, :expression, :failure_message, :warning_message]
```

Comment:
Here maybe, we could remove the `warning_message` from the required list, and even the field from the view if the type is not `expect_enum`.
But this could be an improvement in future PRs as well, out of scope of this current work

Reply:
Let's skip this for now, as it goes a little bit out of scope. But I would track this down